apis/core/v1beta1/openstackversion_types.go

```diff
@@ -166,6 +166,7 @@ type ContainerTemplate struct {
 	TestTobikoImage                   *string `json:"testTobikoImage,omitempty"`
 	TestHorizontestImage              *string `json:"testHorizontestImage,omitempty"`
 	TestAnsibletestImage              *string `json:"testAnsibletestImage,omitempty"`
+	OpenstackLightspeedImage          *string `json:"openstackLightspeedImage,omitempty"`
```

Comment:
I think it would be good to drop the parameter from the OpenStackVersion and Controlplane resources and just have Lightspeed obtain its Image directly from the ENV variable. This would also work in the case where Lightspeed gets installed ahead of the Controlplane if that use case materializes.